{

################# map origin #####################

# [lat,lon,altidude]
map0:[39.449658 , -76.554066 , 0.0],

################# Odometry topic #####################
odometry_topic_name     : /insekf/pose,

################# Allow backward motion of USV? ######
AllowBack : false,

################# Being used with Alvar indoors? ######
isIndoor : true,

################# Filter frequency #####################
hz_odom:  20.0  # Freq at which odometric data is published

}
